Source/SwiftLintFramework/Rules/Lint/BlanketDisableCommandRule.swift

```diff
@@ -0,0 +1,143 @@
+import SwiftSyntax
+
+struct BlanketDisableCommandRule: ConfigurationProviderRule {
+    typealias ConfigurationType = BlanketDisableCommandConfiguration
+
+    var configuration = BlanketDisableCommandConfiguration()
+
+    init() {}
+
+    static let description = RuleDescription(
+        identifier: "blanket_disable_command",
+        name: "Blanket Disable Command",
+        description: "swiftlint:disable commands should be re-enabled before the end of the file",
+        kind: .lint,
+        nonTriggeringExamples: [
+            Example("""
+            // swiftlint:disable unused_import
+            // swiftlint:enable unused_import
+            """),
+            Example("""
+            // swiftlint:disable unused_import unused_declaration
+            // swiftlint:enable unused_import
+            // swiftlint:enable unused_declaration
+            """),
+            Example("// swiftlint:disable:this unused_import"),
+            Example("// swiftlint:disable:next unused_import"),
+            Example("// swiftlint:disable:previous unused_import")
+        ],
+        triggeringExamples: [
+            Example("// ↓swiftlint:disable unused_import"),
+            Example("""
+            // ↓swiftlint:disable unused_import unused_declaration
```

Comment:
Would it be possible to have the marker at the identifier like
```suggestion
            // swiftlint:disable unused_import ↓unused_declaration
```
so I don't have to read the complete violation message and extract the identifier name from there.

Reply:
Yep, that is much better.